<ui version="4.0" >
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure Acquisition</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>Icon-Config.xpm</normaloff>Icon-Config.xpm</iconset>
  </property>
  <property name="toolTip" >
   <string/>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>294</x>
     <y>604</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="outputFileLabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>21</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Output filename:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputFileLE" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>20</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>c:\blah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseBut" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>19</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="acqModeLabel" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>85</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The acquisition mode: regular or INTAN demuxd</string>
   </property>
   <property name="text" >
    <string>Acquisition Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="acqModeCB" >
   <property name="geometry" >
    <rect>
     <x>132</x>
     <y>84</y>
     <width>175</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The acquisition mode: regular or INTAN demuxd</string>
   </property>
   <item>
    <property name="text" >
     <string>AI 60 Demux (4 INTANs)</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Straight AI</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>AI 120 Demux (8 INTANs)</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="channelListLE" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>267</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Enter a list of _real_ AI channels to acquire.  Ranges such as 0-3 or comma-separated-lists such as 1,2,3,4 are ok.</string>
   </property>
   <property name="text" >
    <string>0:3</string>
   </property>
  </widget>
  <widget class="QLabel" name="channelListLabel" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>241</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Enter a list of _real_ AI channels to acquire.  Ranges such as 0:3 or comma-separated-lists such as 1,2,3,4 are ok.</string>
   </property>
   <property name="text" >
    <string>AI Channel List: (ranges/lists of the form: 0:3, 5, 7, 9:10, etc)</string>
   </property>
  </widget>
  <widget class="QLabel" name="srateLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>150</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Sampling rate per (virtual) channel, if externally clocked this is only a reference rate for datafile metadata</string>
   </property>
   <property name="text" >
    <string>Sampling Rate Hz (per channel):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="srateSB" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>212</x>
     <y>150</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Sampling rate per (virtual) channel, if externally clocked this is only a reference rate for datafile metadata</string>
   </property>
   <property name="minimum" >
    <number>10</number>
   </property>
   <property name="maximum" >
    <number>1000000</number>
   </property>
   <property name="value" >
    <number>29630</number>
   </property>
  </widget>
  <widget class="QComboBox" name="clockCB" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>342</x>
     <y>150</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>This property is not directly modifiable, but it indicates the clock mode used. 
Straight AI always uses an internal clock, whereas the MUX modes use an external clock.</string>
   </property>
   <item>
    <property name="text" >
     <string>Ext. clock (PFI2)</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Internal clock</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="disableGraphsChk" >
   <property name="geometry" >
    <rect>
     <x>24</x>
     <y>607</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Disabling graphing of data saves on CPU/GPU load. Graphs can be re-enabled at any time from the system tray.</string>
   </property>
   <property name="text" >
    <string>Disable data graphs (just save to disk)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="acqStartEndGB" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>428</y>
     <width>431</width>
     <height>152</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Specify timed or triggered acquisitions here</string>
   </property>
   <property name="title" >
    <string>Acquisition Start/End</string>
   </property>
   <widget class="QComboBox" name="acqStartEndCB" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>23</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Choose the acquisition trigger type, or Immediate for immediate mode</string>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <item>
     <property name="text" >
      <string>Immediate Start</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Photodiode Controlled Start &amp; End</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Photodiode Controlled Start Only</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Timed</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>StimulateOpenGLII Plugin Start &amp; End</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>StimulateOpenGLII Plugin Start Only</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="acqFrame" >
    <property name="geometry" >
     <rect>
      <x>7</x>
      <y>55</y>
      <width>417</width>
      <height>92</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="acqStartEndDescrLbl" >
     <property name="geometry" >
      <rect>
       <x>15</x>
       <y>9</y>
       <width>381</width>
       <height>69</height>
      </rect>
     </property>
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:10pt; font-style:italic; color:#294928;">The acquisition will start immediately.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="aiRangeMinLbl" >
   <property name="geometry" >
    <rect>
     <x>222</x>
     <y>118</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The AI Range to use for the acquisition</string>
   </property>
   <property name="text" >
    <string>AI Range Min-Max:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="aiRangeCB" >
   <property name="geometry" >
    <rect>
     <x>336</x>
     <y>118</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The AI Range to use for the acquisition</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceLabel" >
   <property name="geometry" >
    <rect>
     <x>23</x>
     <y>119</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The NI Device to use for the acquisition</string>
   </property>
   <property name="text" >
    <string>Device:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="deviceCB" >
   <property name="geometry" >
    <rect>
     <x>78</x>
     <y>119</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The NI Device to use for the acquisition</string>
   </property>
  </widget>
  <widget class="QLabel" name="doCtlLabel" >
   <property name="geometry" >
    <rect>
     <x>312</x>
     <y>86</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>(INTAN Mode only) The DO line to use for INTAN control</string>
   </property>
   <property name="text" >
    <string>DO Ctl:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="doCtlCB" >
   <property name="geometry" >
    <rect>
     <x>372</x>
     <y>84</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>(INTAN Mode only) The DO line to use for INTAN control</string>
   </property>
  </widget>
  <widget class="QComboBox" name="aiTerminationCB" >
   <property name="geometry" >
    <rect>
     <x>119</x>
     <y>179</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The termination setting to use for the AI channels.  See NI docs for an explanation of these terms.  Default is usually good enough.</string>
   </property>
   <item>
    <property name="text" >
     <string>Default</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>RSE</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>NRSE</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Differential</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>PseudoDifferential</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="aiTerminationLabel" >
   <property name="geometry" >
    <rect>
     <x>19</x>
     <y>179</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The termination setting to use for the AI channels.  See NI docs for an explanation of these terms.  Default is usually good enough.</string>
   </property>
   <property name="text" >
    <string>Termination:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fastSettleSB" >
   <property name="geometry" >
    <rect>
     <x>369</x>
     <y>177</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The amount of time in milliseconds to toggle the DO line off/on when performing a "fast settle" of the INTAN MUX.  Ignored in non-INTAN mode.</string>
   </property>
   <property name="minimum" >
    <number>1</number>
   </property>
   <property name="maximum" >
    <number>2000</number>
   </property>
   <property name="value" >
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="fastSettleLbl" >
   <property name="geometry" >
    <rect>
     <x>269</x>
     <y>179</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The amount of time in milliseconds to toggle the DO line off/on when performing a "fast settle" of the INTAN MUX.  Ignored in non-INTAN mode.</string>
   </property>
   <property name="text" >
    <string>Fast Settle MS:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="auxGainLbl" >
   <property name="geometry" >
    <rect>
     <x>21</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The gain setting on the external amplifier.  Incoming sample voltages are divided by this value when graphing to screen.</string>
   </property>
   <property name="text" >
    <string>AUX Gain:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="auxGainSB" >
   <property name="geometry" >
    <rect>
     <x>119</x>
     <y>211</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>The gain setting on the external amplifier.  Incoming sample voltages are divided by this value when graphing to screen.</string>
   </property>
   <property name="minimum" >
    <double>0.010000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000.000000000000000</double>
   </property>
   <property name="value" >
    <double>200.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="muxMapBut" >
   <property name="geometry" >
    <rect>
     <x>265</x>
     <y>205</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text" >
    <string>INTAN MUX Mapping...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stimGLReopenCB" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>52</y>
     <width>435</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p align="justify" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If checked, then whenever a Stimulate OpenGL Plugin (re)starts, the current save file is closed and a new one is opened.  The format for this save file name is:&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">PREFIX_STIMGLPLUGINNAME_N&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p align="justify" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Where PREFIX is the filename specified in the above lineedit, STIMPLUGINNAME is the name of the plugin, and N is an integer number to make the filename unique.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p align="justify" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Note: Checking this requires a copy of StimGL II to be running otherwise data will not be saved.&lt;/p>&lt;/body>&lt;/html></string>
   </property>
   <property name="text" >
    <string>Re-Open New Save File on StimGL Experiment (Re)Start</string>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="aoPassthruWidget" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>298</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>false</bool>
  </property>
  <property name="gridSnapY" >
   <bool>false</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
